probe.h

```diff
@@ -29,8 +29,8 @@
 // Probe pin initialization routine.
 void probe_init();
 
-// Returns probe pin state.
-uint8_t probe_get_state();
+// Returns probe pin state. Triggered = true. Called by gcode parser and probe state monitor.
+#define probe_get_state() (!(PROBE_PIN & PROBE_MASK))
```

Comment:
Not a huge deal, but what is the reasoning for replacing a function with a #define?  You're no inlining this function everywhere it's used.  Is there any danger or benefit?